### PR TITLE
feat(activity): update feed icon styles

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

[Figma](https://www.figma.com/design/PzsUxBv03Kg6GgzsSq5R1D/Mobile-Stack-x-Mento?node-id=2143-22325&node-type=instance&t=rjn4GaF88yEn3LHC-0)

Changes include:
- updates to background and border color of feed item image
- drop network icon badge
- removed default avatar and replace with user icon

### Test plan

https://github.com/user-attachments/assets/03bf134f-bb05-4caa-b151-e54723cae197

Failed: (marked everything as failed for testing purposes)

### Related issues

- Fixes ACT-1434

### Backwards compatibility

Yes

### Network scalability

N/A

### CODE REVIEW

File: src/transactions/feed/NftFeedItem.tsx [hunk 1]
```diff
@@ -35,28 +34,26 @@ function NftFeedItem({ transaction }: Props) {
     <Touchable testID={'NftFeedItem'} disabled={false} onPress={openNftTransactionDetails}>
       <View style={styles.container}>
         {/* Try to show the first image. Otherwise display the default icons */}
-        <IconWithNetworkBadge networkId={transaction.networkId}>
```
Comment (finnianjs) [inline src/transactions/feed/NftFeedItem.tsx:38]: Is this change necessary? We aren't supporting NFTs in mento right so this code shouldn't be reached?

Comment (satish-ravi) [reply src/transactions/feed/NftFeedItem.tsx:38]: yes, not necessary but just did it anyway to be complete

Comment (finnianjs) [reply src/transactions/feed/NftFeedItem.tsx:38]: Should it also be removed from `PositionIcon` and `cardDetails` for completeness, and then comment out / remove component?

**Review by finnianjs — APPROVED** [review]
left one response to my earlier comment but otherwise looks good

File: src/transactions/feed/NftFeedItem.tsx (hunk 1 above)
Comment (satish-ravi) [reply src/transactions/feed/NftFeedItem.tsx:38]: this PR is scoped to tx feed items, we can do that separately if required
